mixer/pkg/api/grpcServer.go

```diff
@@ -216,6 +221,11 @@ var reportResp = &mixerpb.ReportResponse{}
 
 // Report is the entry point for the external Report method
 func (s *grpcServer) Report(ctx context.Context, req *mixerpb.ReportRequest) (*mixerpb.ReportResponse, error) {
+
+	if s.throttler.Throttle(loadshedding.RequestInfo{PredictedCost: float64(len(req.Attributes))}) {
```

Comment:
We want this on Check too, right?

Reply:
I think so. I'm trying to rationalize how best to support multiple thresholds while keeping configuration relatively simple. For instance, we probably want different latency constraints for `Check` than for `Report`. I have a couple of small ideas. When I get a bit more inspiration along that path, I'll update this PR.

Reply:
Why is cost 1.0? Should we instead treat CHECK as high criticality over REPORT?

Reply:
Yes -- but criticality (or QoS) isn't really reflected in the model (as proposed here).

However, given our current deployment architecture, we get the "proper" behavior from this mechanic as-is. `istio-policy` will only handle `Check()` (and `istio-telemetry` only `Report()`). As a result, there shouldn't ever be cross-RPC concerns in the same process.

I think in order to support differing priorities across RPC methods we will ultimately need a more service-config like way of providing instructions. I was trying to keep this (somewhat) simpler to start.

Reply:
Ok. Semantically, this is just estimating the number of dispatched calls (1 vs many for report), so this makes sense.